plugins/amoeba/platforms/common/src/AmoebaCommonKernels.cpp

```diff
@@ -1814,11 +1820,28 @@ void CommonCalcAmoebaGeneralizedKirkwoodForceKernel::initialize(const System& sy
         defines["MUTUAL_POLARIZATION"] = "";
     else if (polarizationType == AmoebaMultipoleForce::Extrapolated)
         defines["EXTRAPOLATED_POLARIZATION"] = "";
+    defines["DIELECTRIC_OFFSET"] = cc.doubleToString(force.getDielectricOffset());
+    tanhRescaling = force.getTanhRescaling();
+    // Max Born radius is 3 nm (30 A).
+    defines["BIG_RADIUS"] = cc.doubleToString(3.0);
+    if (tanhRescaling) {
+        defines["TANH_RESCALING"] = cc.intToString(1);
```

Comment:
That's an interesting way of writing "1". :)  Since the kernel interprets it as a bool, it would be clearer to set it to "true" or "false" rather than "1" or "0".